src/System.Private.Xml/src/System/Xml/Xslt/XslTransform.cs

```diff
@@ -29,7 +28,7 @@ private XmlResolver _DocumentResolver
                 if (_isDocumentResolverSet)
                     return _documentResolver;
                 else
-                    return XsltConfigSection.CreateDefaultResolver();
```

Comment:
There were two `CreateDefaultResolver()` methods, one in `XsltConfigSection` and the other `XmlReaderSection`.

`XsltConfigsection` implementation was:
```
internal static XmlResolver CreateDefaultResolver()		
        {		
            if (LocalAppContextSwitches.s_ProhibitDefaultUrlResolver)		
                return XmlNullResolver.Singleton;		
            else		
                return new XmlUrlResolver();		
        }
```